public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.test.ts

```diff
@@ -569,6 +569,8 @@ describe('transformSceneToSaveModelSchemaV2', () => {
         },
         // Some other properties that aren't in the annotation spec
         // and should be moved to options
+
+        // TODO: in what scenario that would be even possible?! DOESN'T make sense
```

Comment:
@axelavargas - I can't think of a scenario when this test would make sense. Whatever was non-standard property, should be in `legacyOptions`. What scenario were you trying to address here?